05-UI-ReactJS/react-builder-master/source/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Router, Route, hashHistory } from 'react-router';
+import { Provider } from 'react-redux';
+import App from './containers/app';
+import Add from './components/add';
+import List from './components/list';
+import moviesApp from './reducers/reducers';
+
+const moviesStore = createStore(moviesApp)
+const appRoute = document.getElementById('app');
+ ReactDOM.render(
+  <Provider store={moviesStore}>
+    <Router history={hashHistory}>
+      <Route path='/' component={App} />
```

Comment:
If you have routes as nesting you could declare app in just one place and render the forms and list as children of it.. Try to do that